content/docs/tutorials/kubernetes/guestbook.md

```diff
@@ -212,15 +212,15 @@ const frontendService = new k8s.core.v1.Service("frontend", {
         name: "frontend",
     },
     spec: {
-        type: isMinikube ? "ClusterIP" : "LoadBalancer",
+        type: useLoadBalancer ? "ClusterIP" : "LoadBalancer",
```

Comment:
You want to flip the branches here right? i.e. use `LoadBalancer` when `useLoadBalancer` and `ClusterIP` otherwise.